src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown

```diff
@@ -1,122 +1,168 @@
-The left field is the width, and the right field is the height.
+Device Mode offers a wide array of devices for emulation. You can add a 
+custom device if you find an edge-case or niche device that isn't covered. 
 
-#### Retina displays
+<div class="wf-devtools-flex">
+  <div>
+  <p>To add a custom device:</p>
+  <ol>
+    <li>Go to DevTools Settings.</li>
+    <li>Click the <strong>Devices</strong> tab.</li>
+    <li>Click <strong>Add custom device</strong>.</li>
+    <li>Enter a device name, width, height, device pixel ratio, and 
+     user agent string.</li>
+     <li>Click <strong>Add</strong>.</li>
+  </ol>
+  <p>Your custom device is now available in the <strong>Device</strong> dropdown menu.</p>
+  </div>
+  <div class="wf-devtools-flex-half">
+    <img src="imgs/custom-device.png" alt="select a device">
+  </div>
+</div>
+
+### Device states and orientation
+
+![toggle orientation](imgs/change-orientation.png)
+
+When emulating a specific device, the Device Mode toolbar shows an additional control that primarily serves as a way to toggle the orientation between landscape and portrait.
+
+<div class="wf-devtools-flex">
+  <div>
+    <p>On selected devices, the control does more than just orientation toggling. For supported devices like the Nexus 5X, you'll get a dropdown that allows you to emulate certain device states, like:</p>
+    <ul>
+      <li>Default browser UI</li>
+      <li>With Chrome navigation bar</li>
+      <li>With opened keyboard</li>
+    </ul>
+  </div>
+  <div class="wf-devtools-flex-third">
+    <img src="imgs/change-device-state.png" alt="Change the Device UI">
+  </div>
+</div>
+
+### Zoom to fit  
+
+<div class="wf-devtools-flex">
+  <div>
+  <p>Sometimes you'll want to test a device that has a resolution larger than the actual available space in your browser window. In these cases, the <strong>Zoom to Fit</strong> option comes in handy:</p>
+  <ol>
+    <li><strong>Fit to Window</strong> will automatically set the zoom level to the maximum available space.</li>
+    <li><strong>Explicit percentages</strong> are useful if you want to test DPI on images, for instance.</li>
+  </ol>
+  </div>
+  <div class="wf-devtools-flex-third">
+    <img src="imgs/zoom-to-fit.png" alt="Zoom to Fit">
+  </div>
+</div>
+
+## Optional controls (e.g. touch, media queries, DPR)
+
+<div class="wf-devtools-flex">
+  <div>
+  <p>Optional controls can be changed or enabled by clicking on the three little dots on the right side of the device toolbar. Current options include</p>
+  <ul>
+    <li>User agent type (Emulates UA and touch events)</li>
+    <li>Device pixel ratio</li>
+    <li>Media Queries</li>
+    <li>Rulers</li>
+    <li>Configure Network (UA, Network Throttling)</li>
+  </ul>
+  </div>
+  <div class="wf-devtools-flex-third">
+    <img src="imgs/device-mode-dotmenu.png" alt="Device Mode Settings">
+  </div>
+</div>
+
+Read on to learn more about the specific options.
+
+### User agent type (&gt; Chrome 50: Device type)
```

Comment:
This confused me. Are you saying that the name is different depending on what Chrome you're using? E.g. Chrome 49 or below, and Chrome 50 or above

50 lands in stable in only a month. I'd just focus on that and drop references to Chrome 49 and below